backend/modules/cohorts/repository.py

```
@@ -0,0 +1,547 @@
+            CohortStudent.student_id == student_id,
+            CohortStudent.status == "approved"
+        ).first()
+        
+        if not enrollment:
+            return []
+        
+        return self.db.query(CohortSimulation, Scenario).join(
+            Scenario, CohortSimulation.simulation_id == Scenario.id
+        ).filter(
+            CohortSimulation.cohort_id == cohort_id,
+            Scenario.is_draft == False,
```

Ruff (0.14.8)

Avoid equality comparisons to `False`; use `not Scenario.is_draft:` for false checks

Replace with `not Scenario.is_draft`

(E712)